lib/neo4j/shared/permitted_attributes.rb

```diff
@@ -0,0 +1,23 @@
+module Neo4j::Shared
+  module PermittedAttributes
+    extend ActiveSupport::Concern
+    include ActiveModel::ForbiddenAttributesProtection
+
+    def process_attributes(attributes)
+      attributes = sanitize_for_mass_assignment(attributes)
+      super(attributes)
+    end
+
+    def attributes=(attributes)
+      attributes = sanitize_for_mass_assignment(attributes)
+      super(attributes)
+    end
+
+    protected
+
+    def sanitize_for_mass_assignment(attributes)
+      attributes = super(attributes)
+      attributes.respond_to?(:symbolize_keys) ? attributes.symbolize_keys : attributes
```

Comment:
I think that `symbolize_keys` shouldn't be what `sanitize_for_mass_assignment` is responsible for.  Did you move it out so that it would apply for both `ActiveNode` and `ActiveRel`?